src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -263,9 +282,11 @@ public void validateArguments() {
 
         // throw if args were specified for a filter that was also disabled
         disableFilters.forEach(s -> {
-            if (requiredPredecessors.contains(s)) {
+            if (requiredPredecessors.contains(s) && !toolDefaultReadFilters.containsKey(s)) {
                 throw new CommandLineException(
                         String.format("Values were supplied for (%s) that is also disabled", s));
```

Comment:
There doesn't appear to be a test that executes this throw (I'm not sure if it was hit before, but since the if stmt is changed we should have tests that hit both clauses).